content/articles/introduction-to-udoo-x86-ii-ultra/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: engineering-education
+url: /introduction-to-udoo-x86-ii-ultra/
+title: Introduction to Udoo X86 II Ultra session board controller (SBC)
+description: This article describes the built in architecture and processing power of the Udoo x86 II Ultra. It also highlights and clarifies the different capabilities of Udoo X86 II Ultra and why it is better than most session board controllers.
+author: sheldon-aswani
+date: 2021-08-07T00:00:00-19:30
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/introduction-to-udoo-x86-ii-ultra/hero.jpeg
+    alt: Introduction to Udoo X86 II Ultra image
+---
+Session board controllers are extremely important devices as they allow us to run multiple applications usually run on a PC. In this article, we will focus on `Udoo x86 II Ultra`'s capabilities and enhanced features.
+<!--more-->
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Introduction to Udoo X86 II Ultra](#introduction-to-udoo-x86-ii-ultra)
+- [Pros](#pros)
+- [Cons](#cons)
+- [Conclusion](#conclusion)
+
+### Prerequisites
+To follow through this article, you need to have a basic understanding of `session board controllers`
+
+### Introduction to Udoo X86 II Ultra
+Udoo X86 II Ultra session board controller's power is comparable to that of a new typical personal computer in the market today. Udoo X86 II Ultra comes embedded on the same platform with an Arduino Leonardo 101 board made with a more powerful X86 maker architecture.
```

Comment:
Why would you talk about compatibility with Arduino when it has it embedded already?